tests/test_forecast.py

```diff
@@ -7,8 +7,8 @@ def test_run_forecast():
     site = PVSite(latitude=51.75, longitude=-1.25, capacity_kwp=1.25)
 
     # run model with icon and gfs nwp
-    predications_df_gfs = run_forecast(site=site, ts='2023-10-30', nwp_source="gfs")
-    predications_df_icon = run_forecast(site=site, ts='2023-10-30', nwp_source="icon")
+    predications_df_gfs = run_forecast(site=site, ts='2023-12-30', nwp_source="gfs")
+    predications_df_icon = run_forecast(site=site, ts='2023-12-30', nwp_source="icon")
```

Comment:
I wonder if we should to `today- 2 months` so that we dont get this error again?

Reply:
Maybe its a good idea to have both. So one hard coded and one relative like -2 days. When we have the feature to do the predictions for timestamps later then 90 days, this is also a relevant test. But let me add this test to #103 then!

Reply:
Thanks for this, thats a good idea